### ISSUE TITLE
libnpmpublish token not working

### ISSUE THREAD

**Issue by KTibow** [npm/cli#7091]
Even though I pass a token to libnpmpublish, I get 404/429 errors when I try to publish a package. I've tried a variety of types of tokens to no avail. How do I get this to work?

**Comment by KTibow** [comment]
Fixed w/ https://github.com/npm/cli/issues/4250

**Comment by macovedj** [comment]
Does it make sense to add this information to the `libnpmpublish` docs?  I just spent quite a bit of time with these issues as well, and this was the fix.  Not sure under what circumstances it's possible to use `libnpmpublish` w/o the `forceAuth` field, but if it's necessary, a lot of folks could probably save a lot of time if the example were updated.

**Comment by lukekarrys** [comment]
Agreed that a docs update would be helpful here. If anyone wants to make a PR to update the `libnpmpublish` README that would be great!
